src/include/configuration.h

```diff
@@ -131,6 +131,26 @@ extern "C" {
 int
 pgagroal_init_configuration(void* shmem);
 
+/**
+ * Check and set directory path
+ * @param path Pointer to the path to be set
+ * @param directory_path Directory to search for path
+ * @param default_path Default path
+ * @param fallback_path Fallback path if default not found
+ */
+void
+pgagroal_check_directory_path(char** path, const char* directory_path, const char* default_path, const char* fallback_path);
+
+/**
+ * Check and set directory path
```

Comment:
The comment is the same as the previous function, it should be more clear in which way the two are different.